res/css/structures/_ToastContainer.pcss

```diff
@@ -104,21 +105,17 @@ limitations under the License.
             }
         }
 
-        .mx_Toast_title,
-        .mx_Toast_description {
-            padding-right: 8px;
-        }
-
         .mx_Toast_title {
             display: flex;
             align-items: center;
             column-gap: 8px;
             width: 100%;
             box-sizing: border-box;
+            margin-bottom: $spacing-16;
 
             h2 {
```

Comment:
Could use the styled Heading component instead of the h2 tag and remove most of this style

Reply:
Except this style matches our `h3` styling, which might cause a bit more fallout with changed snapshots etc

Reply:
The `mx_Toast_title_countIndicator` below also could be replaced with a `Caption` component